gslib/tests/test_hmac.py

```diff
@@ -18,17 +18,25 @@
 from __future__ import print_function
 from __future__ import unicode_literals
 
-import re
 import boto
+import os
+import re
```

Comment:
Is `re` used anywhere?

Reply:
yep, here for example https://github.com/GoogleCloudPlatform/gsutil/blob/92a16e73cf5dad3cc246a417b5cfd80720849b9f/gslib/tests/test_hmac.py#L75